cmd/localCI/console.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2014,2015,2016 Docker, Inc.
```

Comment:
maybe you can reference where the original code came from, presuming you copied this from runc or similar? Put that comment after the copyright header probably.